client/sites-dashboard/components/sites-site-launch-nag.tsx

```diff
@@ -88,18 +92,18 @@ export const SiteLaunchNag = ( { site }: SiteLaunchNagProps ) => {
 	} );
 
 	const {
-		data: { checklist },
+		data: { checklist, is_dismissed, launchpad_screen },
 		isLoading,
 	} = useLaunchpad( site.slug, getChecklistSlug( site ), undefined, 'sites-dashboard' );
 
-	if ( 'unlaunched' !== site.launch_status || ! checklist || isLoading ) {
+	if ( ! checklist || isLoading || is_dismissed ) {
 		return null;
 	}
 
 	let numberOfSteps = checklist.length || 0;
 	let completedSteps = ( checklist.filter( ( task ) => task.completed ) || [] ).length;
 
-	if ( shouldShowLaunchpadFirst( site ) ) {
+	if ( shouldShowLaunchpadFirst( site ) && launchpad_screen !== 'skipped' ) {
```

Comment:
If you start a new site and skip the focused launch checklist the launch site task will be a task again in /home and the count will mismatch here if we subtract that "Launch site" task from the count. 

cc @p-jackson re https://github.com/Automattic/wp-calypso/pull/101028/files?w=1#r1986443534

Reply:
Sigh, what a mess 😓